feature/danggn/src/main/java/com/mashup/feature/danggn/data/DanggnDao.kt

```diff
@@ -57,4 +58,10 @@ interface DanggnDao {
     suspend fun getDanggnSingleRound(
         @Path("danggnRankingRoundId") danggnRankingRoundId: Int
     ): Response<DanggnRankingSingleRoundCheckResponse>
+
+    @POST("api/v1/danggn/ranking-reward-comment/{danggnRankingRewardId}")
+    suspend fun postDanggnRankingRewardComment(
+        @Path("danggnRankingRewardId") danggnRankingRewardId: Int,
+        @Body scoreRequest: DanggnRankingRewardCommentRequest,
```

Comment:
```suggestion
        @Body commentRequest: DanggnRankingRewardCommentRequest,
```

naming 수정하면 좋을 것 같아요!

Reply:
수정했어요!!

Reply:
엇 commend로 한것같아요 ㅋㅋㅋㅋㅋ comment :)

Reply:
바보.. 다시 수정했습니다 ㅜ